web/api/v1/api.go

```diff
@@ -1868,3 +1894,20 @@ OUTER:
 	}
 	return matcherSets, nil
 }
+
+func parseLimitParam(limitStr string) (limit int, err error) {
+	limit = math.MaxInt
+	if limitStr != "" {
+		limit, err = strconv.Atoi(limitStr)
+		if limitStr == "" {
```

Comment:
How can this happen if you already checked on line 1900?

Reply:
my bad, just resolved it.